assets/scripts/templates/helpers/owner.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
```

Comment:
I don't think this is the way we're going to be able to get i.e. a user's name from the `owner` of a photo. There's no get route for them, and mongoose has its own functionality for this sort of thing.

See: https://mongoosejs.com/docs/populate.html